packages/common/src/models/Track.ts

```diff
@@ -298,6 +302,10 @@ export type TrackMetadata = {
   }
 } & Timestamped
 
+export type WriteableTrackMetadata = TrackMetadata & {
```

Comment:
nice - i like this. it probably should just eventually be a separate field than `artwork`, but this at least much it much more clean / legible